packages/core/src/components/Button/Button.stories.tsx

```diff
@@ -194,3 +194,33 @@ export const Semantic = ({ onClick }) => {
 Semantic.args = {
   onClick: clickAction,
 };
+
+export const SetComponent = ({ onClick }) => {
+  return (
+    <HvBox sx={{ display: "flex", gap: 20, padding: 20 }}>
+      <HvButton onClick={onClick}>Button element</HvButton>
+      <HvButton onClick={onClick} component="a">
```

Comment:
If we're using an `a`, we want an `href` here, not `onClick`

```suggestion
      <HvButton component="a" href="https://example.com">
```